### PR TITLE
Live ads support

### PR DESCRIPTION
**Pull request by incompl**
Depends on #147

### CODE REVIEW

**Comment by misteroneill** [comment]
LGTM 👍

File: src/videojs.ads.js [hunk 1]
```diff
@@ -43,13 +43,35 @@ var
   },
 
   /**
+   * Returns a boolean indicating if given player is in live mode.
+   */
+  isLive = function(player) {
+    if (player.duration() === Infinity) {
+      return true;
+    } else if (videojs.browser.IOS_VERSION === "8" && player.duration() === 0) {
```
Comment (dmlap) [inline src/videojs.ads.js]: @gkatsev Should video.js be handling this discrepancy? Seems like the kind of problem we try to abstract over.

Comment (gkatsev) [inline src/videojs.ads.js]: Sounds like iOS is once again doing stuff that is outside of the spec (https://html.spec.whatwg.org/multipage/embedded-content.html#dom-media-duration) however, yes, that's totally something that videojs itself should be handling.

Comment (incompl) [inline src/videojs.ads.js]: I could open an issue against VJS for this and link to it in a comment. Sound like a plan?

Comment (gkatsev) [inline src/videojs.ads.js]: That works. Or, if you want to make a PR against stable, we can get a patch release out chikchak.

Comment (incompl) [inline src/videojs.ads.js]: Opened issue and added comment

**Pull request merged**